static/js/around-map.js

```diff
@@ -51,45 +52,50 @@ map.on('click', event => {
   if(poi) {
     highlight(poi._id)
     urlFor(poi._id)
-    moveTo([poi.position.longitude, poi.position.latitude])
+    moveTo([poi.position.latitude, poi.position.longitude])
   }
 })
 
 // functions
-function currentMarker (currentLocation) {
+function currentMarker (currentLatLng) {
   const popup = new mapboxgl.Popup({offset: [10, 0]})
     .setText('Your current location')
 
   const el = document.createElement('div')
   el.classList.add('marker')
   el.classList.add('current')
 
-  reverseCoords(currentLocation)
+  // OSM standard [Lng, Lat]
+  const lngLat = [currentLatLng[1], currentLatLng[0]]
   new mapboxgl.Marker(el, {offset:[0, -30]})
-    .setLngLat(reverseCoords)
+    .setLngLat(lngLat)
```

Comment:
I just understood why you were not using `reverse()`. It actually modifies the original array. 🤒
However, because you only use once the flipped array, you could do it directly in the function: `.setLngLat([currentLatLng[1], currentLatLng[0]])`